cyences_app_for_splunk/appserver/static/cs_overview.js

```diff
@@ -71,6 +71,7 @@ require([
         { token: 'db_oracle', associated_products: ['Oracle'] },
         { token: 'db_mssql', associated_products: ['MSSQL'] },
         { token: 'f5_bigip', associated_products: ['F5 BIGIP'] },
+        { token: 'cloudflare', associated_products: ['Cloudflare'] },
```

Comment:
pull latest changes from the master or resolve the merge conflict. this format is changed slightly in recent changes